securesystemslib/signer/_signature.py

```diff
@@ -15,6 +15,12 @@ class Signature:
     Provides utility methods to easily create an object from a dictionary
     and return the dictionary representation of the object.
 
+    Args:
+        keyid: HEX string used as a unique identifier of the key.
+        sig: HEX string representing the signature.
+        unrecognized_fields: Dictionary of all attributes that are not managed
+            by securesystemslib.
```

Comment:
Repeating Args and Attributes seems silly. I'd prefer to only use "Args" here and say *"All parameters named below are not just constructor arguments but also instance attributes."* as we do in the `Key` class.

The problem is we use `sig` as arg name and `signature` as attribute name. I think we should consolidate them now. Changing the (positional) arg name seems less disrupting.

Reply:
yeah, consolidate now sounds good: No-one outside of securesystemslib should should be calling Signature() so the fallout should be minimal.

This doesn't have to be in this PR though